.azure-pipelines.yml

```diff
@@ -54,8 +64,8 @@ jobs:
       - task: Cache@2
         displayName: "Cache vcpkg's packages"
         inputs:
-          key: .\vcpkg.json | "$(VCPKG_REF)" | ".v2"
-          path: "$(VCPKG_DIR)"
+          key: .\vcpkg.json | "$(VCPKG_REF)"
+          path: '%BUILD_DIR%/vcpkg_installed'
```

Comment:
since we're using `vcpkg.json` now it's considered "manifest" mode, which effectively means cmake/vcpkg is figuring out what to install at configuration time, as opposed to "classic" mode where we'd install things manually.

Reply:
and in manifest mode, vcpkg is installing to build/vcpkg_installed. which explains why we didn't have any cache hits anymore..